resources/lib/playerinfo.py

```diff
@@ -17,30 +17,36 @@ def __init__(self, **kwargs):
         self._tracker = None
         self._last_pos = 0
         self._total = None
+        self._stop = False
         Player.__init__(self)
-        while not self._monitor.abortRequested():
-            if self._monitor.waitForAbort(10):
+        while not self._monitor.abortRequested() and not self._stop:
+            if self._monitor.waitForAbort(1):
                 break
+        self._info((self._last_pos, self._total))
 
```

Comment:
What's the purpose of moving the info update here?

Reply:
It was also for testing the behaviour, we should revert this, as it impacts when the container refresh happens. What I discovered is that none of the threads ever stopped, so after playing and stopping 10 videos, the older threads act also on the older videos, so we have to break out of the thread ourselves and that fixes the effect I saw where when I stopped the 10th video, the previous 9 videos would have the same resumetime.

Reply:
At some point I went nuts, as I couldn't figure out what was happening (because I randomly started videos without recollecting which ones I started so not seeing the theme...).